src/sql/workbench/contrib/query/browser/queryInputFactory.ts

```diff
@@ -26,7 +26,7 @@ const editorInputFactoryRegistry = Registry.as<IEditorInputFactoryRegistry>(Edit
 
 export class QueryEditorLanguageAssociation implements ILanguageAssociation {
 	static readonly isDefault = true;
-	static readonly languages = ['sql', 'kusto'];
+	static readonly languages = ['sql', 'kusto'];		//TODOKusto Add language id here for new languages supported in query editor. Make it easier if possible.
```

Comment:
Is this todo being tracked anywhere?

Reply:
@aaomidi this is a private branch we are working on currently. We will try to tackle few before checking it in to master. Depending on how many are left to be tackled, I will create tasks.